modules/configuring-private-storage-endpoint-azure-user-provided-vnet-subnet.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-user-provided-vnet-subnet"]
+= Configuring a private storage endpoint on Azure with user-provided VNet and subnet names
```

Comment:
maybe it's better replace VNet with vnet

Reply:
Thanks. For the time being, I think I'll keep it as VNet because in our docs it's most commonly "VNet". Will talk to the lead writer to see what he thinks about changing all instances in the future.